src/main/java/io/rebloom/client/Client.java

```diff
@@ -154,6 +172,33 @@ public boolean add(String name, byte[] value) {
     }
   }
 
+  public boolean[] bfInsert(String key, String... items) {
+    return bfInsert(key, (InsertOptions) null, (ReserveParams) null, items);
+  }
+
+  public boolean[] bfInsert(String key, InsertOptions insertOptions, ReserveParams reserveParams, String... items) {
+    final List<byte[]> args = new ArrayList<>();
+    args.add(SafeEncoder.encode(key));
+    if (insertOptions != null) {
+      args.addAll(insertOptions.getOptions());
+    }
+    if (reserveParams != null) {
+      args.addAll(reserveParams.getParams());
+    }
+    args.add(Keywords.ITEMS.getRaw());
+    for (String item : items) {
+      args.add(SafeEncoder.encode(item));
+    }
+    try (Jedis conn = _conn()) {
+      List<Object> listResp = sendCommand(conn, Command.INSERT, args).getObjectMultiBulkReply();
+      final int lastIndex = listResp.size() - 1;
+      if (listResp.get(lastIndex) instanceof JedisDataException) {
```

Comment:
@gkorland To be able to return `List<Boolean>`. Otherwise, we'd have to return `List<Object>` where last element is JedisDataException and all other elements are Booleans.